PowerShellTools/DebugEngine/ScriptDebugProcess.cs

```diff
@@ -107,8 +136,7 @@ public int GetProcessId(out Guid pguidProcessId)
         public int GetAttachedSessionName(out string pbstrSessionName)

```

Comment:
MSDN: "This method is deprecated, and its implementation should always return E_NOTIMPL."
https://msdn.microsoft.com/en-us/library/vstudio/bb146718(v=vs.100).aspx

(and yes, just realized that says return, not throw, next iteration will have that be fixed)